nc-times/src/components/TopicsBar.jsx

```diff
@@ -0,0 +1,49 @@
+import TopicBarStyle from "../resources/TopicBarStyle"
+import { getArticleByTopic } from "../api"
+
+function TopicsBar(props){
+const {articles, setArticles} = props
+
+    function handleClickCoding(articles){
+   getArticleByTopic("coding").then((data)=>{
+    setArticles(data)
+
+   })
+    }
+    function handleClickFootball(articles){
```

Comment:
These three functions aren't very DRY - think about how you could create one function that can handle the topic dynamically.

Reply:
Also - our topics shouldn't be hardcoded. What if a new topic gets added? What's the most reliable way to get hold of the latest list of topics?